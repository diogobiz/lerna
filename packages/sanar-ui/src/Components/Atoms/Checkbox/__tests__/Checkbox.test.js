import React from "react";
import TestRenderer from "react-test-renderer";

import ESCheckbox from "../";

it("renders correctly", () => {
	const component = <ESCheckbox>Manter logado</ESCheckbox>;

	const tree = TestRenderer.create(component).toJSON();
	expect(tree).toMatchSnapshot();
});
