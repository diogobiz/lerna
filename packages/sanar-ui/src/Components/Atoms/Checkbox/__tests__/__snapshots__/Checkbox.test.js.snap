// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<label
  className="es-checkbox ant-checkbox-wrapper"
>
  <span
    className="ant-checkbox"
    style={Object {}}
  >
    <input
      checked={false}
      className="ant-checkbox-input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      type="checkbox"
    />
    <span
      className="ant-checkbox-inner"
    />
  </span>
  <span>
    Manter logado
  </span>
</label>
`;
